qiskit/result/mitigation/local_readout_mitigator.py

```diff
@@ -91,6 +91,11 @@ def __init__(
                 ainv = np.linalg.pinv(mat)
             self._mitigation_mats[i] = ainv
 
+    @property
+    def settings(self) -> Dict:
+        """Return settings."""
+        return {"assignment_matrices": self._assignment_mats, "qubits": self._qubits}
+
```

Comment:
Why not adding it to the super class `BaseReadoutMitigator`?

Reply:
This is since the assignment matrix is a 2^n-by-2^n matrix (where n is the number of qubits), so it will not be possible to generate it for advanced mitigation methods (such as M3).